app/swagger/requests/user.rb

```diff
@@ -44,7 +44,7 @@ class User
                 property :in_progress_forms
                 property :account, type: :object do
                   property :account_uuid,
-                           type: %w[string null], # while Account feature is still flagged we need to allow null
+                           type: %w[string null],
```

Comment:
Leaving `null` in here b/c technically this value still could be `null`, in the off chance a user does not have an `Account` record.